packages/services/src/models/notification-scanners/AppealsOpened.js

```diff
@@ -0,0 +1,47 @@
+import NotificationScannerBaseModel from './NotificationScannerBaseModel'
+import Network from '@aragonone/court-backend-server/build/web3/Network'
+
+class AppealsOpened extends NotificationScannerBaseModel {
+  async scan() {
+    let notifications = []
+    const fourDaysBeforeNow = Math.floor((Date.now()-(4*this._DAYS))/1000)
+    const query = `
+    {
+      adjudicationRounds(where: {stateInt_in: [1,2,3], createdAt_lt: ${fourDaysBeforeNow}}, orderBy: createdAt) {
+        id
+        dispute {
+          id
+        }
+        jurors {
```

Comment:
Should all jurors get AppealsOpened notifications? Should I ignore ones with missed vote/reveal?

Reply:
I think it's fine to send a notification to all the drafted jurors even if they fail to commit or reveal their vote